api/scripts/team-acces/update-fwb-authentication-complements.js

```diff
@@ -0,0 +1,89 @@
+import url from 'url';
+
+import _ from 'lodash';
+import bluebird from 'bluebird';
+
+import { disconnect } from '../../db/knex-database-connection.js';
+import { checkCsvHeader, parseCsvWithHeader } from '../helpers/csvHelpers.js';
+import { usecases } from '../../lib/domain/usecases/index.js';
+import { FWB } from '../../lib/domain/constants/oidc-identity-providers.js';
+
+const MODULE_PATH = url.fileURLToPath(import.meta.url);
+const IS_LAUNCHED_FROM_CLI = process.argv[1] === MODULE_PATH;
+const TIMER_NAME = '';
+const REQUIRED_FIELD_NAMES = ['SUB', 'Firstname', 'Lastname', 'EmployeeNumber', 'Population'];
```

Comment:
```suggestion
const REQUIRED_FIELD_NAMES = ['SUB', 'FirstName', 'LastName', 'EmployeeNumber', 'Population'];
```
Petit typo sur les labels. Le script ne fonctionne pas avec le fichier de test 😄 
Mise à jour aussi à faire ici : https://github.com/1024pix/pix/pull/7681/files#diff-f065b60709d3d3fdeeec3765c8410581f9ad543ffae991a1837229a65e2fff8dR46-R47

Reply:
J'ai fait un fixup avec les modif